docs/UserGuide.md

```diff
@@ -405,7 +409,55 @@ Remarks: Prices will plateau out in a few days.
 ________________________________________________________________________________
 ```
 
-### 2.6.0 Removing an instrument record: `delete`
+### 2.6.0 Aborting an operation: `abort`
+
+Abort a current process. If you want to terminate the `add` or `edit` operation at any stage, enter `abort`
+and the operation will be terminated.
+
+**Note:**  `abort` is case-insensitive but must be entered without any additional characters. For example, 
+`abort/` will be an invalid input.
+
+Aborting an Add Operation:
+
+**Example usage**
+```
+mTracker$add> abort
+```
+**Expected outcome:**
+
+```
+mTracker$main> add
+Please key in the type of instrument:
+mTracker$add> stock
+Name of stock:
+mTracker$add> abort
+Addition of new instrument has been aborted! You are in the main workspace now.
+```
+
+Aborting an Edit Operation:
+
+**Example usage**
+
+```
+mTracker$edit> abort
+```
+
+**Expected outcome:**
+
+```
+mTracker$main> edit 2
+	Please enter one or more StocEtf parameters to edit separated by spaces only.
```

Comment:
```suggestion
	Please enter one or more Etf parameters to edit separated by a single space only.
```

Reply:
I change this to stock instead, because the parameters displayed below are for stock.